docker-compose.yml

```diff
@@ -5,10 +5,8 @@ services:
      IP: ${REDIS_CLUSTER_IP}
      SENTINEL: ${REDIS_USE_SENTINEL}
      STANDALONE: ${REDIS_USE_STANDALONE}
-    build:
-      context: .
-      args:
-        redis_version: '6.2.1'
+    # We are building the image via `make build` which uses buildx since we need it build an multi-arch image.
+    image: grokzen/redis-cluster:latest
```

Comment:
The downside of this move is that in the case you just download this compose version and you just run it w/o running make build before it is that you will download the latest version from docker-hub automatically. The 2 issues with downloading from there these days is the number of downloads permitted by users from docker-hub itself. And the second one is that if you do not specify a version but `:latest` then you might get redis 7 or 8 in the future, and not a reliable redis 6.2.1 like before this.

Reply:
Good points! I've switched this to the custom tag `local` instead. This will cause the build to fail (after seeing the "please run `make build`" message) when the image doesn't exist locally.

Reply:
I could live with that yes